### PR TITLE
Add SES Client

### PR DESCRIPTION
**Pull request by MancunianSam**

### CODE REVIEW

**Review by TomJKing — COMMENTED** [review]
Looks good. Is it worth adding some unit tests for the new SES client? Also the README will need updating with details of the new SES Client

**Comment by MancunianSam** [comment]
Good point on the unit test. I've added one in. I've updated the README as well. I've put something in there about the SQS stuff as well because I didn't update the README whenever I did that.

File: README.md [hunk 1]
```diff
@@ -4,5 +4,12 @@ This is a repository for useful methods and circe decoders. These can be used wh
 
 * S3 Event Decoder - The messages we get through  in a Lambda are an S3 message inside an SNS message inside an SQS message. This method returns all S3 events from the message along with a list of errors, if any.
 * AWS Decoders - Circe decoders for working with the above classes which are used by the event decoder method. These are needed because circe won't auto decode case classes.
+* SQS Utils - There is a send and a delete method for sqs messages.
+* SES Utils - There is a send email method.
+
+There are also clients for AWS services. These are configured with an http client, the region and and an endpoint configurable in an aplication.conf file. There are:
```
Comment (TomJKing) [inline README.md]: Double "and": "... the region and and an endpoint ..."

File: README.md [hunk 2]
```diff
@@ -4,5 +4,12 @@ This is a repository for useful methods and circe decoders. These can be used wh
...
+* An SQS Client
+* An S3 Client
+* As SES Client
```
Comment (TomJKing) [inline README.md]: Should be "An" not "As"

**Review by TomJKing — COMMENTED** [review]
Just a couple of typos in the README

File: README.md (hunk 1 above)
Comment (MancunianSam) [reply README.md]: I think I was getting warnings about that before I switched them off thinking I know better :smile:

File: README.md (hunk 2 above)
Comment (MancunianSam) [reply README.md]: I've changed this one as well.

**Review by TomJKing — APPROVED** [review]

**Pull request merged**